package.json

```diff
@@ -0,0 +1,99 @@
+{
+  "name": "kubevirt-web-ui",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@storybook/addon-actions": "^3.4.10",
+    "@storybook/addon-options": "^3.4.10",
+    "@storybook/react": "^3.4.10",
+    "autoprefixer": "7.1.6",
+    "babel-core": "6.26.0",
+    "babel-eslint": "7.2.3",
+    "babel-jest": "20.0.3",
+    "babel-loader": "7.1.2",
+    "babel-preset-react-app": "^3.1.1",
+    "babel-runtime": "6.26.0",
+    "case-sensitive-paths-webpack-plugin": "2.1.1",
+    "chalk": "1.1.3",
+    "css-loader": "0.28.7",
+    "dotenv": "4.0.0",
+    "dotenv-expand": "4.2.0",
+    "eslint": "4.10.0",
+    "eslint-config-react-app": "^2.1.0",
+    "eslint-loader": "1.9.0",
+    "eslint-plugin-flowtype": "2.39.1",
+    "eslint-plugin-import": "2.8.0",
+    "eslint-plugin-jsx-a11y": "5.1.1",
+    "eslint-plugin-react": "7.4.0",
```

Comment:
how about using the patternfly eslint rules?

Reply:
While contributing to patternfly-react project, I found them over-complicated. The resulting syntax is confusing to me. But this is just my opinion, I can get used to whatever syntax rules anyone with stronger opinion prefers.

Reply:
the main value is that we should work across multiple repos (pf-react, console and this) - it would make life a lot easier to have similar linting configurations....

Reply:
@ohadlevy This is just the initial setup based on create-react-app template to get us started. There will be follow-up changes to simplify project's dependencies and align them with other projects you've mentioned. This also applies to webpack & Babel config etc.

We can indeed try using [eslint-plugin-patternfly-react](https://github.com/patternfly/patternfly-react/tree/master/packages/eslint-plugin-patternfly-react) later on, I've no experience with it myself though. If we end up not using it, we'll still want to get as much overlap as possible to ensure the code looks similar to other projects.

Reply:
@vojtechszocs I understand and support this initiative, some things are easier to remove than others, overall, don't see my comments as blocking merging this.

I do think it does make sense to use the pf-react eslint plugin to avoid large refactoring later on.